Makefile

```
@@ -1,25 +1,34 @@
 .PHONY: build
-build: build-dar build-jscodegen build-ui
+build: build-dar build-triggers-dar build-jscodegen build-ui
 
 
 ### DAR ###
 
-DAR=target/app.dar
+MAIN_DAR=target/supplychain.dar
```

Minor, however I think it's better:
```suggestion
MODELS_DAR=target/supplychain.dar
```

---

ok